util/asn1.c

```diff
@@ -0,0 +1,1122 @@
+/* asn1.c
```

Comment:
You might consider exposing this through some optional API in the library that can optionally be enabled at build-time. Then it would be easier for wolfCLU to consume. This util application can still exist. Although instead of adding another new directory consider putting in examples or scripts.

Reply:
What part goes into the library?
PrintAllAsn1() and functions called?